app/api/spotify/route.ts

```diff
@@ -1,21 +1,25 @@
 import { NowPlaying, Providers } from "@bolajiolajide/now-playing";
 
-export const dynamic = 'force-dynamic'
+export const dynamic = "force-dynamic";
```

Comment:
What does this do?

Reply:
Trying to prevent that page from being pre-rendered ~and eventually caching by Vercel.~